playbooks/roles/ocp-operator-api/defaults/main.yaml

```diff
@@ -0,0 +1,11 @@
+---
+
+op_api_role_enable: false
+build_operator_image: true
+deploy_operator: false 
+op_cleanup: false
+OPERATOR_SDK_VERSION: "1.37.0"
+op_api_dir: "/tmp/operator-api"
+ocp_version: "418"
+quay_username: ""
```

Comment:
1. These are marked as required, so don't set in defaults.
2. Setting empty `quay_username` does not make sense, won't work.